ui/cap-react/src/antd/admin/components/Customize.js

```diff
@@ -52,69 +52,52 @@ const Customize = props => {
   };
 
   return (
-    <Space direction="vertical" style={{ width: "100%" }}>
-      <PropertyKeyEditorForm
-        schema={props.schema && props.schema.toJS()}
-        uiSchema={props.uiSchema && props.uiSchema.toJS()}
-        formData={props.schema && props.schema.toJS()}
-        onChange={_debounce(_onSchemaChange, 500)}
-        optionsSchemaObject="optionsSchema"
-        optionsUiSchemaObject="optionsSchemaUiSchema"
-        title="Schema Settings"
-      />
-
-      <PropertyKeyEditorForm
-        schema={props.schema && props.schema.toJS()}
-        uiSchema={props.uiSchema && props.uiSchema.toJS()}
-        formData={props.uiSchema && props.uiSchema.toJS()}
-        onChange={_debounce(_onUiSchemaChange, 500)}
-        optionsSchemaObject="optionsUiSchema"
-        optionsUiSchemaObject="optionsUiSchemaUiSchema"
-        title="UI Schema Settings"
-      />
-
-      {props._path.size == 0 && (
-        <Card title="UI Options">
-          <Space direction="vertical" size="large" style={{ width: "100%" }}>
-            <Space>
-              <Typography.Text>Size Options</Typography.Text>
+    <Tabs centered style={{ height: "100%", overflow: "scroll" }}>
+      <Tabs.TabPane tab="Schema Settings" key="1">
+        <PropertyKeyEditorForm
+          schema={props.schema && props.schema.toJS()}
+          uiSchema={props.uiSchema && props.uiSchema.toJS()}
+          formData={props.schema && props.schema.toJS()}
+          onChange={_debounce(_onSchemaChange, 500)}
+          optionsSchemaObject="optionsSchema"
+          optionsUiSchemaObject="optionsSchemaUiSchema"
+        />
+      </Tabs.TabPane>
+      <Tabs.TabPane tab="UI Schema Settings" key="2">
+        {props._path.size != 0 ? (
+          <PropertyKeyEditorForm
+            schema={props.schema && props.schema.toJS()}
+            uiSchema={props.uiSchema && props.uiSchema.toJS()}
+            formData={props.uiSchema && props.uiSchema.toJS()}
+            onChange={_debounce(_onUiSchemaChange, 500)}
+            optionsSchemaObject="optionsUiSchema"
+            optionsUiSchemaObject="optionsUiSchemaUiSchema"
+          />
+        ) : (
+          <Space direction="vertical" style={{ padding: "0 12px" }}>
+            <Typography.Text>Size Options</Typography.Text>
+            <Radio.Group
+              size="small"
+              onChange={e => sizeChange(e.target.value)}
+              style={{ paddingBottom: "15px" }}
+            >
               {SIZE_OPTIONS.map(size => (
-                <Tag
-                  onClick={() => sizeChange(size)}
-                  key={size}
-                  color={
-                    props.uiSchema &&
-                    props.uiSchema.toJS()["ui:options"] &&
-                    props.uiSchema.toJS()["ui:options"].size == size &&
-                    "geekblue"
-                  }
-                >
-                  {size}
-                </Tag>
+                <Radio.Button value={size}>{size}</Radio.Button>
```

Comment:
do these work?

Reply:
I'm trying to make them work (they were also not working well before)

Reply:
Done